databases/week1/db_w1.sql

```diff
@@ -0,0 +1,55 @@
+-- 1 Find out how many tasks are in the task table
+select count(id)
+from task;
+
+-- 2 Find out how many tasks in the task table do not have a valid due date
+select count(id)
+from task
+where due_date IS null;
+
+
+-- 3 Find all the tasks that are marked as done (what should I do with null response?)
```

Comment:
Would you please solve this by the JOIN operator?
Thank you

Reply:
I have improved theses exercises. Thanks for your feedback :)